dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClasspathChecker.java

```diff
@@ -92,54 +102,49 @@ public static void main(String[] arguments)
 
     CommandLine commandLine = ClasspathCheckOption.readCommandLine(arguments);
     ImmutableList<Path> inputClasspath = ClasspathCheckOption.generateInputClasspath(commandLine);
-    // TODO(suztomo): take command-line option to choose entry point classes for reachability
-    ImmutableSet<Path> entryPoints = ImmutableSet.of(inputClasspath.get(0));
 
-    boolean onlyReachable = commandLine.hasOption("r");
-    ClasspathChecker classpathChecker = create(onlyReachable, inputClasspath, entryPoints);
+    ImmutableSet.Builder<Path> entryPoints = ImmutableSet.builder();
```

Comment:
I feel like `ClassPathCheckOption` should do this work to determine the entry points.

Reply:
It makes sense. Updated.